pkgs/by-name/zi/zipline/package.nix

```
@@ -87,18 +87,15 @@ stdenv.mkDerivation (finalAttrs: {
     openssl
   ];
 
-  YARN_ENABLE_TELEMETRY = "0";
-
-  ZIPLINE_DOCKER_BUILD = "true";
+  env = environment // {
+    YARN_ENABLE_TELEMETRY = "0";
+    ZIPLINE_DOCKER_BUILD = "true";
+  };
```

```suggestion
  env = {
    YARN_ENABLE_TELEMETRY = "0";
    ZIPLINE_DOCKER_BUILD = "true";
  } // environment;
```
otherwise we cannot override those variables

---

Thx, done